Chap_API_Tools.tex

```diff
@@ -313,6 +313,14 @@ \subsection{Direct launch}
     \item \pasteAttributeItem{PMIX_DEBUG_WAIT_FOR_NOTIFY}
 \end{itemize}
 
+
+\adviceuserstart
+The \refattr{PMIX_IOF_FILE_ONLY} indicates output is directed to files and
+no copy is sent back to the application.  For example, this can be combined with
+\refattr{PMIX_IOF_OUTPUT_TO_FILE} or \refattr{PMIX_IOF_OUTPUT_TO_FILE} to
```

Comment:
One of these should be "output_to_directory", yes?

Reply:
LOL.  Yes

Reply:
Just updated the branch based on your comments - minus this change. You may need to rebase (I don't think so, but possible).

Reply:
OK, fixed typo.  I think it should merge to your branch cleanly.  If problems, please just copy/paste.  Thanks Ralph.